### ISSUE TITLE
xtb parameters up to Z=120 ?

### ISSUE THREAD

**Issue by miroi** [grimme-lab/xtb#669]
Hello,
would you please consider extending parameters up to Z=120 to cover also superheavy elements with xTB ?

**Comment by awvwgk** [comment]
What is the application of having elements beyond 86 with a non-relativistic valence-only electronic structure method?

**Comment by miroi** [comment]
Ah, good point. I thought that for high Z you have some relativity already incorporated via empirical parameters. But this is not the case. So, closing this ticket. Thanks for your answer.

**Comment by awvwgk** [comment]
I think the closest possible to a “relativistic” xTB Hamiltonian might be the non-colinear spin-polarized calculation + spin-orbit coupling in DFTB+. I tested the non-colinear version already, but haven't checked whether I already allowed the spin-orbit code path.

Deriving or fitting the parameters (element parameters including f-shells, spin constants, spin-orbit coupling constants) is another challenges which requires reference data at an appropriate level. Technical this might actually be possible.

**Comment by miroi** [comment]
Hello,

I am reopening this issue. xtb is very fast method and we are insterested in having it for elements up to Z=120.

**Comment by dipayanster** [comment]
Indeed, having superheavy elements in xTB will be very much helpful. 

I am interested to know if there is any progress about this.

**Comment by malrusdjav** [comment]
we need superheavy elements

**Comment by awvwgk** [comment]
Happy to continue this discussion on the feasibility of a valence only non-relativistic tight binding method for super heavy elements.

What kind of systems do you have in mind for using tight binding here?

**Comment by miroi** [comment]
> f a valence only non-relativistic tight binding method for super heavy elements.
> 
> What kind of systems do you have in mind for using tight binding here?

Hello Sebastian,

with superheavy elements, we study molecular and periodic systems like those in https://www.tandfonline.com/doi/full/10.1080/00268976.2023.2293229 or https://pubs.acs.org/doi/abs/10.1021/acs.inorgchem.6b02759 . 

Would be good to have xtb covering also this "relativistic" realm.

**Comment by thfroitzheim** [comment]
The next xTB method will be parametrized up to Lr (Z=103) but a further extension from our side is unlikely. Already there, a meaningful fit set and parametrization are very challenging (needs diverse geometries and electronic states) and the DFT reference becomes questionable and hard to converge. Thus, parametrization for even heavier elements seems not very promising from our perspective.
